docs/DeveloperGuide.md

```diff
@@ -463,6 +463,27 @@ As the user will not be able to remember the link to our User Guide, we hyperlin
 The following activity diagram summarizes what happens when a user executes a `edit` command for `ApplicationStatus` and `InterviewStatus` <br>
 <img src="images/HelpWindowActivityDiagram.png" width="250" />
 
+### Focus Feature
+
+#### What is this feature about?
+The 'focus' feature utilises the different panels in our application. It will bring up the intended Candidate's information, 
+such as course, email into the middle panel for the user to see. 
+
+#### How is this feature implemented?
+This feature is implemented with the similar implementation of `CandidateListCard`, as we utilise the JavaFX codes to 
+create the `view`. We also introduced new features such as `ImageView`, where the extra information will consist of a display
+picture for the Candidate (WIP). 
```

Comment:
created a placeholder issue for working on how the display picture will be uploaded (if relevant)! #182

Reply:
Oki !! Thanks Charmaine !